pandas/io/excel/_openpyxl.py

```diff
@@ -389,6 +389,18 @@ def _convert_to_protection(cls, protection_dict):
 
         return Protection(**protection_dict)
 
+    @classmethod
+    def _to_excel_range(cls, startrow, startcol, endrow, endcol):
```

Comment:
Rather than using this can you just pass a CellRange into openpyxl?

Reply:
Yes, but doesn't the CellRange constructor only accept a string of the form `A1:B2`?

Reply:
I think can use `min_col`, `min_row`, `max_col`, `max_row` from constructor

https://openpyxl.readthedocs.io/en/stable/api/openpyxl.worksheet.cell_range.html#openpyxl.worksheet.cell_range.CellRange

Reply:
indeed, the cell range can be constructed from those arguments, however the Table constructor itself really only accepts a string. Internally also the string representation is used, so I don't see a way around this.

Reply:
I think can just cast to a str then:

```python
>>> from openpyxl.worksheet.cell_range import CellRange
>>> str(CellRange(min_col=1, min_row=1, max_col=3, max_row=3))
'A1:C3'
```

That's what we are ultimately after here right? (save maybe off-by-one)

Reply:
That was what I expected, but I was on openpyxl 2.4.8, and this apparently a 2.5.0 feature. Assuming updating the minimum supported version is not an issue I will fix this